src/komponenter/kalkulator/utregninger.test.ts

```diff
@@ -26,14 +30,60 @@ test('Finn dato en dag tilbake fra angitt dato', () => {
     );
 });
 
+export const testFunksjonAvTidslinje = (tidsLinje: DatoMedKategori[]) => {
+    let bestårTest = true;
```

Comment:
Bra du fikset feilen og la til test 😄 Noen kommentarer. Jeg forstår ikke helt hva denne testen gjør 🤔 Er det mulig å isolere det tilfelle du fant feilen i, og kjøre testen bare for dette caset? 
Og legg gjerne koden inn i selve testen `test('dette skal skje')`, med et navn som er enda tydeligere på hva som testes - "tidslinje riktig konstruert" kan bety mye!